tmt/utils.py

```diff
@@ -4839,6 +4839,7 @@ def field(
         envvar: Optional[str] = None,
         deprecated: Optional['tmt.options.Deprecated'] = None,
         help: Optional[str] = None,
+        show_default: Optional[str] = None,
```

Comment:
Isn’t it rather a `bool`, without `Optional`?

Reply:
yeah, was thinking where the problem is and then rather went for lunch 😆